schema/dataKeys.json

```diff
@@ -192,6 +200,10 @@
                       "type": "string"
                     }
                   },
+                  "component": {
+                    "type": "string",
+                    "description": "The component of the image e.g. adv-comp-1"
```

Comment:
This had to be added in see https://redhat-internal.slack.com/archives/C04J47GL936/p1729590287898049?thread_ts=1729520292.052339&cid=C04J47GL936